AcadeMeData/tests.py

```diff
@@ -45,50 +45,46 @@ def test_get_user(self):
         assert User.get_user('username') == user_for_example.user
 
 
-class TestUniversityModel(pytest):
-    @pytest
+@pytest.mark.django_db
+class TestUniversityModel:
+    def university_example(self):
+        university = University(university_id=1, name='Reichman University', location="Herzlia",
+                                description="A nice place")
+        university.save()
+        return university
+
     def test_get_university_by_name(self):
-        test_university = University(name='BS').save()
-        universityTest = University.get_university_by_name(
-            'BS')
-        assert test_university == universityTest
-        assert universityTest.exist()
+        test_university = self.university_example()
+        university_test = University.get_university_by_name(
+            'Reichman University')
+        assert test_university == university_test
+        assert isinstance(university_test, University)
 
-    @pytest
     def test_get_university_by_location(self):
-        test_university = University(location='Jerusalem').save()
-        universityTest = University.get_university_by_location(
-            'Jerusalem')
-        assert test_university == universityTest
-        assert universityTest.exist()
+        test_university = self.university_example()
+        university_test = University.get_university_by_location(
+            'Herzlia')
+        assert test_university == university_test
+        assert isinstance(university_test, University)
 
 
+@pytest.mark.django_db
 class TestProffesorModel:

```

Comment:
Note here that you wrote "proffesor"
not a big deal, but if you want you can change it to "professor"

Reply:
RESOVLED